### PR TITLE
SPU2: default to XAudio when failing to read conf

### PR DESCRIPTION
**Pull request by GovanifY**
also switched fprintf to Console.Warning

### CODE REVIEW

**Comment by lightningterror** [comment]
Forgot linux.

**Comment by GovanifY** [comment]
@lightningterror If you look at the diff this bit of code was originally changed because I purged DSound from the Windows implem. I doubt I removed DSound from Linux too, so behaviour should stay consistent with past revisions.

**Comment by lightningterror** [comment]
I was talking about adding a fallback to linux too, but it already has one so scratch what I said.

File: pcsx2/SPU2/Windows/Config.cpp [hunk 1]
```diff
@@ -154,7 +154,9 @@ void ReadSettings()
 	if (mods[OutputModule] == nullptr)
 	{
 		// Unsupported or legacy module.
-		fwprintf(stderr, L"* SPU2: Unknown output module '%s' specified in configuration file.\n", omodid);
+		Console.Warning("* SPU2: Unknown output module '%s' specified in configuration file.\n", omodid);
+		Console.Warning("* SPU2: Defaulting to XAudio (%S).\n", XAudio2Out->GetIdent());
```
Comment (lightningterror) [inline pcsx2/SPU2/Windows/Config.cpp:158]: Is new line at the end needed? Console already does that.

Comment (GovanifY) [reply pcsx2/SPU2/Windows/Config.cpp]: good catch!

**Review by lightningterror — APPROVED** [review]

**Pull request merged**